Extension/browser/chrome/lib/background-page.js

```diff
@@ -25,104 +25,91 @@ var browser = browser || chrome;
         BaseEvent.apply(this, arguments);
     };
 
-    TabEvent.prototype = {
-        __proto__: BaseEvent.prototype
-    };
+    Object.setPrototypeOf(TabEvent.prototype, BaseEvent.prototype);
```

Comment:
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/setPrototypeOf

Warning: Changing the [[Prototype]] of an object is, by the nature of how modern JavaScript engines optimize property accesses, a very slow operation, in every browser and JavaScript engine. The effects on performance of altering inheritance are subtle and far-flung, and are not limited to simply the time spent in obj.__proto__ = ... statement, but may extend to any code that has access to any object whose [[Prototype]] has been altered. If you care about performance you should avoid setting the [[Prototype]] of an object. Instead, create a new object with the desired [[Prototype]] using Object.create().